index.html

```diff
@@ -192,6 +192,50 @@ <h2 class="hf">About Me</h2>
           </ul>
         </div>
       </section>
+      <!-- form -->
+      <section class="contacts">
+        <p class="hg">
+          I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.
+        </p>
+        <form action="" class="getcontact">
+              <!-- <label for="name" class="name"></label> -->
+              <input class="input"  type="text" name="name" placeholder="Full name" required/>
+              <!-- <label for="mail" class="mail"></label> -->
+              <input  class="input" type="email" name="email" placeholder="Email address" required>
+              <!-- <label for="msg"></label> -->
+              <textarea cols="30" rows="10" class="textarea" name="message" placeholder="Enter text here" required></textarea>
```

Comment:
Please consider adding left and right margins to the input tags and the text area field. It's a project requirement to make your design exactly in accordance with the template you have chosen.

## Solution:--

```
.class_name {
  margin-left: 24px;
  margin-right: 24px;
}

```

## Your Design:-

![image](https://user-images.githubusercontent.com/94223367/220681280-2bbccb46-9dd3-4d69-bd64-d9e7e5fd42ec.png)


## Figma Design:-

![image](https://user-images.githubusercontent.com/94223367/220681462-fb064a36-4a3d-4bd4-aabf-9d36c0036b06.png)